### ISSUE TITLE
Image preview sometimes doesn't work without window resize

### ISSUE THREAD

**Issue by 21cygnus12** [sxyazi/yazi#1521]
### What system are you running Yazi on?

Linux Wayland

### What terminal are you running Yazi in?

foot

### `yazi --debug` output

```Shell
Yazi
    Version: 0.3.1 (Arch Linux 2024-08-17)
    Debug  : false
    OS     : linux-x86_64 (unix)

Ya
    Version: 0.3.1 (Arch Linux 2024-08-17)

Emulator
    Emulator.via_env: ("foot", "tmux")
    Emulator.via_csi: Ok(Foot)
    Emulator.detect : Foot

Adapter
    Adapter.matches: Sixel

Desktop
    XDG_SESSION_TYPE: Some("wayland")
    WAYLAND_DISPLAY : Some("wayland-1")
    DISPLAY         : Some(":0")

SSH
    shared.in_ssh_connection: false

WSL
    /proc/sys/fs/binfmt_misc/WSLInterop: false

Variables
    SHELL              : Some("/usr/bin/zsh")
    EDITOR             : Some("nvim")
    VISUAL             : Some("nvim")
    YAZI_FILE_ONE      : None
    YAZI_CONFIG_HOME   : None

Text Opener
    default: Some(Opener { run: "${EDITOR:-vi} \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })
    block  : Some(Opener { run: "${EDITOR:-vi} \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })

Multiplexers
    TMUX               : true
    tmux version       : 3.4
    ZELLIJ_SESSION_NAME: None
    Zellij version     : No such file or directory (os error 2)

Dependencies
    file             : 5.45
    ueberzugpp       : No such file or directory (os error 2)
    ffmpegthumbnailer: No such file or directory (os error 2)
    magick           : No such file or directory (os error 2)
    fzf              : 0.54.3
    fd               : 10.1.0
    rg               : 14.1.0
    chafa            : No such file or directory (os error 2)
    zoxide           : 0.9.4
    7z               : No such file or directory (os error 2)
    7zz              : No such file or directory (os error 2)
    jq               : No such file or directory (os error 2)


--------------------------------------------------
When reporting a bug, please also upload the `yazi.log` log file - only upload the most recent content by time.
You can find it in the "/home/zack/.local/state/yazi" directory.
```

### Did you try the latest nightly build to see if the problem got fixed?

No, and I'll explain why below

### Describe the bug

For the explanation of why i didn't try using nightly, it's because I'm unsure if this is a bug or simply a configuration issue, as I only installed yazi for the first time 5 minutes ago. 

Basically, some of my .jpg images work fine with the image preview, other .jpg's don't work in when the window is maximized, but I found that if i shrink the window the preview will eventually show up. 

### Minimal reproducer

Not sure what to put here, all I did was open my Pictures folder and select an image.

**Comment by sxyazi** [comment]
I didn't quite understand what you meant, could you record a video to show me what's happening? Also, have you tried [setting the `max_width`](https://yazi-rs.github.io/docs/image-preview#size)?
